react/components/ProductImages/Wrapper.js

```diff
@@ -20,12 +20,18 @@ const ProductImagesWrapper = props => {
     [props.images, valuesFromContext.selectedItem]
   )
 
+  const videos = useMemo(() => props.videos != null
+    ? props.videos
+    : path(['videos'], valuesFromContext.selectedItem || {}) || [],
+  [props.videos, valuesFromContext.videos])
+
   return (
     <ProductImages
       zoomProps={props.zoomProps}
       position={props.position || props.thumbnailPosition} //thumbnailPosition is a legacy prop from product-details
       displayThumbnailsArrows={props.displayThumbnailsArrows}
       images={images}
+      videos={videos}
```

Comment:
Is this a video url?

Reply:
Actually it's an object with video props. For now we have only `videoUrl` in this object, but it can be improved later. See this as well https://github.com/vtex-apps/store-resources/pull/44/files

Reply:
Alright, thanks!